### DISCUSSION TITLE
Offline Map Material: Map preview before download?

### DISCUSSION THREAD

**Discussion by DIrgi** [devemux86/cruiser#484] [Q&A] [answered]
Hi Emux, first of all you really do a great job., thank you very much
Yesterday I even recognized that you are the manager behind the Mapforge map material - respect.

As you probably recognized I just started to work with your map. In general I'm on the way switching from a pure navigation hardware (Garmin) to a smartphone or even tablet driven navigation this has several reasons. Anyway, so I do a lot of testing at the moment with my tablet hardware and navigation apps - but so far I'm really satisfied.
 
Anyway since we are here in the idea section, I have a idea or proposal to have a kind of a preview map displayed, before it comes in the App to a map download for offline usage.

I where checking "Freizeitkarte" and "Openandromap", they all have small previews of the area the map is dealing with - honestly I don't know if you have this for mapforge as well. Here are some examples from Freizeitkarte or Openandromap:

**Alps+** (on AndroMaps)
![image](https://github.com/devemux86/cruiser/assets/57968526/5832562a-f5ee-4a16-a529-b91437f4e610)

or 
**Western, Eastern, Northern, Southern Germany** (on Freizeitkarte)
Ok Freizeitkarte is a little more tricky since they show (like here in the picture) sometimes one picture for several maps.
But OK, when there's a download for southern Germany and I see this picture and when I'm a little bit aware about geographic, I know which part of the displayed map is must be.

However I think this could be very helpful (and might save download volume) to see the real map coverage before especially when the maps don't cover an exact bordered country area
![image](https://github.com/devemux86/cruiser/assets/57968526/30ab6751-208f-4eb2-a831-e12444aa33ed)

**Comment by devemux86** [comment, marked as answer]
@DIrgi thanks for the suggestion.

I am the developer of [Mapsforge / VTM](https://github.com/mapsforge?tab=repositories) libraries and maps (for many years).

Cruiser currently offers offline maps from 3 providers: Mapsforge, OpenAndroMaps, Freizeitkarte.
In Mapsforge server we create the maps with the structure from [Geofabrik](https://download.geofabrik.de/) regions.

Creating a different interface for the map downloader requires work and is in my list for the future.
Usually the features that not exist yet have priority to offer them first to the users. :slightly_smiling_face:

**Reply by DIrgi** [reply]
fair enough, cheers 🙂
